chart/templates/deployment.yaml

```
@@ -106,6 +106,12 @@ spec:
         runAsNonRoot: true
       serviceAccountName: {{ include "chart.fullname" . }}-controller-manager
       terminationGracePeriodSeconds: 10
+      {{- if .Values.controller.nodeSelector }}
+        {{- toYaml .Values.controller.nodeSelector | nindent 6 }}
+      {{- end }}
+      {{- if .Values.controller.tolerations }}
+        {{- toYaml .Values.controller.tolerations | nindent 6 }}
+      {{- end }}
```

probably need to include the top level keys.
```suggestion
      {{- if .Values.controller.nodeSelector }}
      nodeSelector:
      {{- toYaml .Values.controller.nodeSelector | nindent 8 }}
      {{- end }}
      {{- if .Values.controller.tolerations }}
      tolerations:
      {{- toYaml .Values.controller.tolerations | nindent 8 }}
      {{- end }}
```